contrib/epee/src/string_tools.cpp

```diff
@@ -112,81 +116,70 @@ namespace string_tools
   
   std::string num_to_string_fast(int64_t val)
   {
-    /*
-    char  buff[30] = {0};
-    i64toa_s(val, buff, sizeof(buff)-1, 10);
-    return buff;*/
     return boost::lexical_cast<std::string>(val);
   }
-	
-	
-    bool compare_no_case(const std::string& str1, const std::string& str2)
-	{
-		
-		return !boost::iequals(str1, str2);
-	}
-	//----------------------------------------------------------------------------
-	 std::string& get_current_module_name()
-	{
-		static std::string module_name;
-		return module_name;
-	}
-	//----------------------------------------------------------------------------
-	 std::string& get_current_module_folder()
-	{	
-		static std::string module_folder;
-		return module_folder;
-	}
-	
+  
+  bool compare_no_case(const std::string& str1, const std::string& str2)
+  {
```

Comment:
Again more whitespace changes. These lines will now attribute you, even though nothing much as been done.

Summoning @moneromooo-monero who used to push back on these.

Reply:
Returned all the non-code changes to the previous state. Thanks.